src/qsSimpleKPI.less

```diff
@@ -1,138 +1,228 @@
+  @import "../semantic/src/definitions/views/statistic.less";
+  @import "../semantic/src/definitions/elements/segment.less";
+
+  .ui.segments, .ui.cards {
+    overflow: auto;
+
+    .segment, .card {
+      min-width: fit-content !important;
+      padding: 0;
+
+      &>.label {
+        display: block;
+        padding: 10px 4px;
+        border-bottom: solid 1px rgb(166,166,166);
+      }
+
+      &>.statistics {
+        padding: 1em;
+        box-sizing: border-box;
+      }
+
+      &.is-selected > .label {
+        background-color: #52cc52;
+        color: rgb(255,255,255);
+      }
+    }
+  }
+
+  .top-aligned-items {
+    height: 100%;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .bottom-aligned-items {
+    height: 100%;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .center-aligned-items {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .stretched-items {
+    height: 100%;
+    display: flex;
+    align-items: stretch;
+  }
+
+  .ui.statistics, .ui.cards, .ui.segments  {
+    width: 100%;
+  }
+
+  .ui.vertical.segments  {
+    height: 100%;
+  }
+
+  @media (max-width: 767px) {
+    .ui.cards {
+      height: 100%;
+    }
+  }
+
+  .ui.statistics .statistic > .label,
+  .ui.statistic > .label {
+    text-transform: none;
+  }
+
+  .ui.segment > .label[class*="center aligned"] {
+    text-align: center;
+  }
+
+  .ui.segments, .ui.segment {
+    background-color: transparent !important;
+  }
+
+  .ui.statistics .statistic > .value,
+  .ui.statistic > .value {
+    text-transform: none;
+  }
+
+  .ui.cards {
+    margin: 0 !important;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .ui.cards > .card,
+  .ui.card {
+    padding: 0;
+    background-color: transparent !important;
+  }
+
+  .ui.tag.label {
+    font-size: 0.5em !important;
+  }
+
+  .ui.button {
+    cursor: pointer;
+    display: inline-block;
+    min-height: 1em;
+    outline: 0;
+    border: none;
+    vertical-align: baseline;
+    background: #fff;
+    color: rgba(0,0,0,.87);
+    font-family: icons,'Helvetica Neue',Arial,Helvetica,sans-serif;
+    margin: 0 0 1rem;
+    padding: 1em;
+    text-transform: none;
+    text-shadow: none;
+    font-weight: 700;
+    line-height: 1.4285em;
+    font-style: normal;
+    text-align: center;
+    text-decoration: none;
+    border-radius: .28571429rem;
+    box-shadow: 0 1px 2px 0 rgba(34,36,38,.15);
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    transition: 'default';
+    will-change: '';
+    -webkit-tap-highlight-color: transparent;
+  }
+
+  .ui.mini.button, .ui.mini.buttons .button, .ui.mini.buttons {
+    font-size: .71428571em;
+  }
+
+  .ui.icon.button, .ui.icon.buttons .button {
+    padding: 1em;
+  }
+
+  .ui.icon.button>.icon, .ui.icon.buttons .button>.icon {
+    opacity: .9;
+    margin: 0;
+    vertical-align: top;
+  }
+
+  .ui.segments, .ui.cards{

```

Comment:
Really picky, but there's a space missing before { to keep it consistent (this line and a couple selectors below). Should possibly add a csslint aswell (new issue in that case)